src/components/pollutant-exposures/air-quality-slider.tsx

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+
+import { Typography, withStyles, Slider, Toolbar, ListItem, List, WithStyles } from "@material-ui/core";
+import { styles } from "./air-quality-slider.styles";
+import strings from "../../localization/strings";
+import { RouteTotalAirQuality } from "../../types";
+
+/**
+ * Interface describing component props
+ */
+interface Props extends WithStyles<typeof styles> {
+  routeTotalExposures: RouteTotalAirQuality[];
+}
+
+/**
+ * Interface describing component state
+ */
+interface State {}
+
+/**
+ * Component which displays air quality per pollutant in sliders list
+ */
+class AirQualitySlider extends React.Component<Props, State> {
+
+  /**
+   * Component constructor
+   *
+   * @param props props
+   */
+  constructor(props: Props) {
+    super(props);
+    this.state = {};
+  }
+  
+  /**
+   * Render method to display air quality for route sliders
+   */
+  public render = () => {
+    const { routeTotalExposures } = this.props;
+
+    if (routeTotalExposures.length === 0) {
+      return <></>;
+    }
+
+    var maxPollution = Math.round(Math.max(...routeTotalExposures.flatMap(e => e.pollutionValue)));
+    var sliders = this.getSlidersList(maxPollution, routeTotalExposures)
```

Comment:
```suggestion
    const maxPollution = Math.round(Math.max(...routeTotalExposures.flatMap(e => e.pollutionValue)));
    const sliders = this.getSlidersList(maxPollution, routeTotalExposures);
```
Please don't use var or let if the value of the variable is not going to be manually changed later. Const is the right way